pkg/debugapi/chequebook.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package debugapi
+
+import (
+	"math/big"
+	"net/http"
+
+	"github.com/ethersphere/bee/pkg/jsonhttp"
+)
+
+var (
+	errCantChequebookBalance = "Cannot get chequebook balance"
```

Comment:
Consider: `errChequebookBalance`. It's clear enough IMO.

Reply:
Error string should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings